source/api/image/3.0/index.md

```diff
@@ -652,40 +622,75 @@ The following shows a simple use of each of these properties:
 }
 ```
 
-More complex examples are given in the [Complete Response Example](#complete-response).
 
-### 5.5. Related Services
+### 5.6. Extra Functionality
+
+The JSON response also contains properties that describe additional functionality available via the Image API service.
 
 | Property   | Required? | Description |
 | ---------- | --------- | ----------- |
-| `service`  | Optional  | The `service` property provides hooks for additional information to be included in the image description, for example a link to an authentication service. The value _MUST_ be a list of objects. |
+| `extraQualities` | Optional | A list of strings that can be used as the quality parameter, in addition to the ones specified in the referenced profile. |
+| `extraFormats` | Optional | A list of strings that can be used as the format parameter, in addition to the ones specified in the referenced profile. |
+| `extraFeatures` | Optional | A list of strings identifying features supported by the service, in addition to the ones specified in the referenced profile. These strings are defined either in the [table][features-table] below or by [registering an extension][image-extensions]. |
 {: .api-table}
```

Comment:
Probably OK, but we may want to revisit these when we look at https://github.com/IIIF/api/issues/1373